source/administration/production-notes.txt

```diff
@@ -740,20 +777,36 @@ accommodate Windows' commitment rules during peak database use.
 MongoDB on Virtual Environments
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This section describes considerations when running MongoDB in some of the
-more common virtual environments.
+This section describes considerations when running MongoDB in some of 
+the more common virtual environments.
 
 For all platforms, consider :ref:`virtualized-disks-scheduling`.
 
-EC2
-```
+:abbr:`AWS (Amazon Web Services)` :abbr:`EC2 (Elastic Compute Cloud)`
+`````````````````````````````````````````````````````````````````````
+
+- When available, enable :abbr:`AWS (Amazon Web Services)`
+  `Enhanced Networking <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enabling_enhanced_networking>`_ 
+  for your instance. Not all instance types support Enhanced Networking. 
+
+  To learn more about Enhanced Networking, see to the 
+  `AWS documentation <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enabling_enhanced_networking>`_.
+
+- Use provisioned IOPS for the storage, with separate devices for
+  journal and data. Do not use the ephemeral storage available on some
+  instance types as their performance changes moment to moment.
+
+- Disable hyperthreading, 
+  :abbr:`DVFS (dynamic voltage and frequency scaling)`, 
```

Comment:
Are you using the DVFS abbreviation somewhere?

Reply:
It is staged [here](https://docs-mongodbcom-staging.corp.mongodb.com/anthonysansone/DOCSP-1271/administration/production-notes.html#aws-ec2). I was looking for "frequency scaling" and only came across this explanation, which was in sync with what was discussed. Did I miss something?

Reply:
The staging view helped. I see now. 

Do you want more details on some of these. There are linux command line options for some of the stuff, although they are on the default linux ami now.  We have notes on the DVFS stuff also. 

For c states the linux boot option is processor.max_cstate=1 or sometimes intel_idle.max_cstate=1 depending on kernel version. I found this in my notes which helped me at the time: 
https://gist.github.com/wmealing/2dd2b543c4d3cff6cab7 and https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/processor_state_control.html
We tested with c3 and c4 instances. The c and p state controls only worked on the c4 generation, but the c3 ended up being lower noise. 

There's also an idle=poll setting to similar effect. That is baked into the basic ami image now and we're using it.

Reply:
Yes.